controllers/ControllerAdmin.php

```diff
@@ -96,13 +101,13 @@ public function checkForm($params){
 
 			if ($value != preg_replace('/\s+/', '', $value)){
 				$space = true;
-				echo $this->twig->render('profile.twig', ['space' => $space, 'logged'=> self::LOGGED, 'user'=> $_SESSION['user']['pseudo'], 'value'=> $_SESSION['user']['email']]);
+				echo $this->twig->render('profile.twig', ['space' => $space, 'logged'=> self::LOGGED, 'user'=> $key['pseudo'], 'value'=> $key['email']]);
```

Comment:
Codacy found an issue: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$this'.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=5982252)